data/json/snippets/addiction.json

```
@@ -78,27 +85,45 @@
   {
     "type": "snippet",
     "category": "addict_nicotine_mild_asleep",
-    "text": [ "You could use some nicotine." ]
+    "text": [
+      "You could use some nicotine.",
+      "Dreams of a long conversation with a dead relative, a cigarette in their hand and smoke wafts through your brain."
```

This reads weird. Did you mean to write

```suggestion
      "You dream of a long conversation with a dead relative, a cigarette in their hand and smoke wafts through your brain."
```